Wrappers/Python/ccpi/astra/processors/__init__.py

```diff
@@ -5,4 +5,7 @@
 from .AstraBackProjectorMC import AstraBackProjectorMC
 from .AstraForwardProjector3D import AstraForwardProjector3D
 from .AstraBackProjector3D import AstraBackProjector3D
-from .FBP import FBP
\ No newline at end of file
+from .FBP import FBP
+from .FDK_flex import FDK_flex
```

Comment:
agreed with @gfardell to rename `FDK_flex` to `FDK` and at a later stage (probably) merge into a generic `FBP` processor that will be able to treat cone and parallel beam.

Also add warning in `FBP` that it will not be able to treat center of rotation shifts.